server/http.go

```diff
@@ -88,7 +91,15 @@ func runSecretScanStandalone(writer http.ResponseWriter, request *http.Request)
 		return
 	}
 	fmt.Printf("Secret Scan triggered for %s: ", req.ImageNameWithTag)
-	res, err := scan.ExtractAndScanImage(req.ImageNameWithTag)
+
+	scanCtx := scan.NewScanContext("")
```

Comment:
Open to discussion but I am not 100% sure we want status update for operations happening outside gRPC
The status update is only needed for console/gRPC

Reply:
Agree... we can work on the non-grpc workflow later.